pipeline/misc/summary_stats_merger.ipynb

```diff
@@ -164,7 +159,7 @@
    },
    "source": [
     "```\n",
-    "sos run ./summary_stats_merger.ipynb --cwd data --yml_path data/template.yml\n",
+    "sos run ./summary_stats_merger.ipynb --cwd data --yml_path data/template.yml --keep_ambiguous=True\n",
```

Comment:
`--keep_ambiguous=True` is wrong. Should be just `--keep-ambiguous`. To not keep ambiguous variants, use `--no-keep-ambiguous` please document this as well.